PreFreeSurfer/scripts/ACPCAlignment.sh

```diff
@@ -83,38 +104,87 @@ echo "date: `date`" >> $WD/log.txt
 echo " " >> $WD/log.txt
 
 ########################################## DO WORK ########################################## 
-
+if [[ "$SPECIES" != "Human" ]] ; then
+  fslmaths "$Reference2mm" "$WD"/Reference
+  fslmaths "$Reference2mm" -mas "$Reference2mmMask" "$WD"/ReferenceBrain
+fi
```

Comment:
Give a more useful error than "file not found" (if someone makes a bad edit to the pipelines).  Also, document this requirement in the `--ref2mm` and `--ref2mmmask` descriptions.
```suggestion
if [[ "$SPECIES" != "Human" ]] ; then
  if [[ "$Reference2mm" == "" || "$Reference2mmMask" == "" ]]
  then
    log_Err_Abort "--ref2mm and --ref2mmmask are required when species is not 'Human'"
  fi
  fslmaths "$Reference2mm" "$WD"/Reference
  fslmaths "$Reference2mm" -mas "$Reference2mmMask" "$WD"/ReferenceBrain
fi
```